oops/oops.go

```diff
@@ -469,3 +470,61 @@ func Recover(p interface{}) error {
 	}
 	return wrapf(fmt.Errorf("recovered panic: %v", p), "")
 }
+
+// MultiError acts as a container to capture multiple errors and their tracebacks
+// When this object is written to a log it will log all the errors it has been fed.
+type multiError struct {
+	errors []error
+}
+
+func (omr *multiError) Error() string {
+	var b strings.Builder
+	b.WriteString("the following errors occurred:\n\n")
+	for _, e := range omr.errors {
+		if oerr, ok := e.(*oopsError); ok {
+			oerr.writeStackTrace(&b)
+		} else {
+			b.WriteString(e.Error())
+		}
+		b.WriteString("\n\n---\n\n")
+	}
+
+	return b.String()
+}
+
+// Return the slice of errors contained within a MultiError instance, or a slice containing
```

Comment:
Nit: In Go, by convention, documentation begins with the name of the method (this may also impact the ability to successfully generate Go docs from source code comments at https://godoc.org/github.com/samsarahq/go/oops):

```suggestion
// Errors returns the slice of errors contained within a multiError instance, or a slice containing
```